### DISCUSSION TITLE
Contributor License Agreement (CLA) tracking

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#11] [Q&A]
https://github.com/commonhaus/foundation-draft/blob/main/governance/ip-policy-guidance.md#1-licensing says projects can use CLAs.
If they do so, who is maintaining track of the signed CLAs? How does that impact leadership transition?

**Comment by ebullient** [comment]
In my simplistic view, tracking CLA is partly using tools like this: https://github.com/cla-assistant/cla-assistant

In the case of a single-vendor project that requires a CLA (for whatever reason), I think this foundation and the Council would have to work out an agreement as part of on-boarding. It definitely would impact leadership transition, and I think this could be a situation where the foundation can help with resolving that if/when a transition to new project owners occurs. OpenJS does this, and I plan to borrow/learn from their methods to make this work.

**Comment by ebullient** [comment]
Related to #84 and #86
